src/app/components/JumpTo/index.stories.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+import JumpTo from '.';
+import { StoryArgs } from '../../models/types/storybook';
+import metadata from './metadata.json';
+import readme from './README.md';
+
+interface Props {
+  subheadlines?: { heading: string }[];
+}
+
+const Component = ({ subheadlines = [] }: Props) => {
+  const jumpToHeadings = subheadlines.map(subheadline => ({
+    heading: subheadline.heading,
+  }));
+
+  return <JumpTo jumpToHeadings={jumpToHeadings} />;
+};
```

Comment:
Ah, this was not quite what I meant...

```suggestion
interface Props {
  jumpToHeadings?: { heading: string }[];
}

const Component = ({ jumpToHeadings = [] }: Props) => {
  return <JumpTo jumpToHeadings={jumpToHeadings} />;
};
```